jwst/cube_build/ifu_cube.py

```diff
@@ -685,14 +680,13 @@ def setup_IFUCube(self,j):
         err_cube = np.zeros((naxis3, naxis2, naxis1))
 
         IFUCube = datamodels.IFUCubeModel(data=data, dq=dq_cube, err=err_cube, weightmap=idata)
-
         IFUCube.update(self.input_models[j])
-
         IFUCube.meta.filename = self.output_name
+        
+        IFUCubeData.blend_output_metadata(self,IFUCube)
```

Comment:
I see `IFUCube` referenced in preceding lines of the code, but I don't see `IFUCubeData`, at least not in the diffs displayed here. Does `IFUCubeData` exist somewhere?